tests/sqlite/test_lite_api.py

```diff
@@ -218,7 +218,7 @@ def test_lite_api_collection_delete_empty(orb, User):
 @requires_lite
 def test_lite_api_collection_has_record(orb, User):
     users = User.all()
-    assert users.hasRecord(User.byUsername('bob'))
+    assert users.has(User.byUsername('bob'))
```

Comment:
sqlite is a file based SQL system.  It supports most base SQL flows, basic syntax, but to a limited extent.  It is not a production quality database (for web apps) but useful for application based databases (since it is builtin to python and file based).  It is great for documentation because you don't need to setup a whole database.